pkg/controller/networkpolicy/validate_test.go

```diff
@@ -1586,8 +1638,101 @@ func TestValidateAntreaPolicy(t *testing.T) {
 					},
 				},
 			},
+			operation:      admv1.Create,
 			expectedReason: "protocol IGMP does not support Pass or Reject",
 		},
+		// Update use same validate function as create. Only provide one update case here.
```

Comment:
It doesn't really use same validation function but looks repeating the same code in its own function. If they are same, we should reduce the redundancy.

Reply:
Good suggestion. Merged duplicate code blocks.